apps/src/templates/studioHomepages/CoursesTeacherEnglish.jsx

```diff
@@ -33,6 +34,8 @@ class CoursesTeacherEnglish extends Component {
     return (
       <div>
         <div>
+          <ProfessionalLearningSkinnyBanner />
+
```

Comment:
Do we need this extra line here?

Reply:
I added the extra line since there are lines in between the other components in the file — it's cut off here in this PR, but if you view the whole file you can see.